app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/drawer/ExoAppCenterDrawer.vue

```diff
@@ -312,7 +322,12 @@ export default {
 
     handleFileUpload() {
       const MAX_FILE_SIZE = 100000;
+      const imageTypeIndex = 6;
       if (this.$refs.image.files.length > 0) {
+        const imageFormatType = this.$refs.image.files[0].type.substring(imageTypeIndex);
```

Comment:
No need to add the substring operation. You can check directly if the type includes SVG format

Reply:
Yes, now that i remember i was about to check on some image format (a giving list) but then it turns out that only svg format is not working so i forgot to change this part. This is a backport it is already merged on develop how should we proceed ?

Reply:
we will proceed without changing it :)